Elasticsearch:  Add documentation for supported Elasticsearch query transformations (#33072)
grafana/grafana: docs/sources/datasources/elasticsearch.md

Querying and displaying log data from Elasticsearch is available in [Explore]({{< relref "../explore" >}}), and in the [logs panel]({{< relref "../panels/visualizations/logs-panel.md" >}}) in dashboards.
Select the Elasticsearch data source, and then optionally enter a lucene query to display your logs.

> When switching from a Prometheus or Loki datasource in Explore your query will be automatically translated to an Elasticsearch log query with a correct Lucene filter.

Reviewer: ```suggestion
When switching from a Prometheus or Loki data source in Explore, your query is automatically translated to an Elasticsearch log query with a correct Lucene filter.
```
